main.py

```diff
@@ -299,14 +328,17 @@ def wait_until_visible(driver, xpath=None, class_name=None, duration=10000, freq
     parser.add_argument("--screenshot-path", default=None)
     parser.add_argument("--html-path", default=None)
     parser.add_argument("--page-load-timeout", type=int, default=2)
-    parser.add_argument("--driver-type", default="firefox", choices=("firefox", "chrome"))
+    parser.add_argument("--driver-type", default="firefox",
+                        choices=("firefox", "chrome"))
     parser.add_argument("--headless", action="store_true")
     parser.add_argument("--select-payment", action="store_true")
     parser.add_argument("--purchase", action="store_true")
     parser.add_argument("--num-retries", type=int, default=1)
     parser.add_argument("--dont-quit", action="store_true")
-    parser.add_argument("--shoe-type", default="M", choices=("M", "F", "W", "Y", "C"))
+    parser.add_argument("--shoe-type", default="M",
+                        choices=("M", "F", "W", "Y", "C"))
     parser.add_argument("--webdriver-path", required=False, default=None)
+    parser.add_argument("--cvv", required=False)
```

Comment:
I think you screwed something up when you resolved conflicts. You're missing
`--shipping-address` and `--shipping-option`. Also `--cvv` has changed a bit in this PR. Does this file even need to be changed at all if you're just adding a separate GUI?

Reply:
The main.py wont need to be modified at all. I already updated my main.py to the latest PR. Right now i'm in the process of adding all the GUI code. The reason those are missing is because when I was looking at @alfredog1976 PR I only focused on adding the CVV since that's what I needed for my drops and at the time it wasn't pushed to the master yet.